hastexo/migrations/0010_add_user_foreign_key.py

```diff
@@ -1,10 +1,29 @@
 from django.conf import settings
-from django.db import migrations, models
+from django.db import migrations, models, OperationalError
+from django.db.models.base import ObjectDoesNotExist
 import django.db.models.deletion
 
 
 class Migration(migrations.Migration):
 
+    def check_stacks(apps, schema_editor):
+        Stack = apps.get_model("hastexo", "Stack")
+        AnonymousUserId = apps.get_model("student", "AnonymousUserId")
+        stacks_with_no_user = []
+        for stack in Stack.objects.all():
+            try:
+                stack.learner = AnonymousUserId.objects.get(
+                    anonymous_user_id=stack.student_id).user
+            except ObjectDoesNotExist:
+                stacks_with_no_user.append(stack.name)
+        if len(stacks_with_no_user) > 0:
+            raise OperationalError(
+                'Unable to link stacks to users; please make sure that '
+                'the following stacks have a student_id that corresponds '
+                f'to a real user account: {stacks_with_no_user}.'
```

Comment:
Micro-nit: this needs whitespace after the the full-stop.

Reply:
Thanks, done.